conf/mig-agent-conf.go.inc

```diff
@@ -43,13 +43,21 @@ var EXTRAPRIVACYMODE = false
 // disabled at run-time using a config option or command line flag
 var SPAWNPERSISTENT = true
 
+// The directory the agent will look for persistent module configuration files
+// in.
+//
+// XXX This should be improved to take into account Windows paths, but at this
+// time persistent module support is not available on Windows. The agent will
+// attempt to locate a configuration using the module name, e.g., modulename.cfg.
+var PERSISTCONFIGDIR = "/etc/mig"
```

Comment:
Does this need to be persistent module specific, or could it be used by any module? If so, should it be called `MODULESCONFDIR` instead?